--- lxde-common-0.5.5.old/lxpanel/panel.in	2011-07-29 18:04:03.000000000 +0400
+++ lxde-common-0.5.5/lxpanel/panel.in	2012-06-04 21:48:11.004650720 +0400
@@ -8,15 +8,22 @@
     widthtype=percent
     width=100
     height=26
-    transparent=0
-    tintcolor=#000000
-    alpha=0
+    RoundCorners=0
+    RoundCornersRadius=7
+    transparent=1
+    tintcolor=#3f3f3f
+    alpha=255
+    autohide=0
+    heightwhenhidden=2
     setdocktype=1
     setpartialstrut=1
     usefontcolor=1
     fontcolor=#ffffff
-    background=1
-    backgroundfile=@prefix@/share/lxpanel/images/background.png
+    UseFontSize=0
+    FontSize=10
+    background=0
+    backgroundfile=/usr/share/lxpanel/images/background.png
+    iconsize=24
 }
 
 Plugin {
@@ -29,7 +36,7 @@
 Plugin {
     type = menu
     Config {
-        image=@prefix@/share/lxde/images/lxde-icon.png
+        image=/usr/share/icons/rosa-button-lxde.png
         system {
         }
         separator {
@@ -53,8 +60,17 @@
             id=pcmanfm.desktop
         }
         Button {
+            id=lxterminal.desktop
+        }
+        Button {
             id=firefox.desktop
         }
+        Button {
+            id=catfish.desktop
+        }
+        Button {
+            id=lxde-control-center.desktop
+        }
     }
 }
 
@@ -68,8 +84,10 @@
 Plugin {
     type = wincmd
     Config {
+        image=window-manager
         Button1=iconify
         Button2=shade
+        Toggle=0
     }
 }
 
@@ -81,10 +99,6 @@
 }
 
 Plugin {
-    type = pager
-}
-
-Plugin {
     type = space
     Config {
         Size=4
@@ -96,46 +110,117 @@
     expand=1
     Config {
         tooltips=1
-        IconsOnly=0
-        AcceptSkipPager=1
+        ShowIconsTitles=Both
+        FallbackIcon=xorg
         ShowIconified=1
         ShowMapped=1
         ShowAllDesks=0
-        UseMouseWheel=1
+        ShowUrgencyAllDesks=1
         UseUrgencyHint=1
         FlatButton=0
+        ColorizeButtons=0
+        IconThumbnails=0
+        DimmIconified=1
         MaxTaskWidth=150
         spacing=1
+        Mode=Group
+        GroupFoldThreshold=2
+        FoldThreshold=10
+        GroupBy=Class
+        ManualGrouping=0
+        UnfoldFocusedGroup=0
+        ShowSingleGroup=0
+        ShowCloseButtons=0
+        SortBy1=Timestamp
+        SortBy2=Timestamp
+        SortBy3=Timestamp
+        SortReverse1=0
+        SortReverse2=0
+        SortReverse3=0
+        RearrangeTasks=0
+        Button1Action=RaiseIconify
+        Button2Action=Shade
+        Button3Action=Menu
+        ScrollUpAction=PrevWindow
+        ScrollDownAction=NextWindow
+        ShiftButton1Action=Iconify
+        ShiftButton2Action=Shade
+        ShiftButton3Action=Close
+        ShiftScrollUpAction=PrevWindowInCurrentGroup
+        ShiftScrollDownAction=NextWindowInCurrentGroup
+        MenuActionsTriggeredBy=Press
+        OtherActionsTriggeredBy=Click
+        MouseOverAction=None
+        HighlightModifiedTitles=1
+        UseGroupSeparators=0
+        GroupSeparatorSize=0
+        UseXNetWmIconGeometry=0
+        UseXWindowPosition=0
+    }
+}
+
+Plugin {
+    type = dirmenu
+    Config {
+        path=XDG_DOCUMENTS_DIR
+        ShowHidden=0
+        ShowFiles=1
+        MaxFileCount=10
+        ShowFileSize=0
+        ShowIcons=1
+        SortDirectoriesBy=Name
+        SortFilesBy=Name
+        PlainView=0
+    }
+}
+
+Plugin {
+    type = dirmenu
+    Config {
+        path=XDG_DOWNLOAD_DIR
+        ShowHidden=0
+        ShowFiles=1
+        MaxFileCount=10
+        ShowFileSize=0
+        ShowIcons=1
+        SortDirectoriesBy=Name
+        SortFilesBy=Name
+        PlainView=0
+    }
+}
+
+Plugin {
+    type = xkb
+    Config {
+        DisplayType=1
+        PerAppLayout=1
+        DefaultGroup=0
     }
 }
 
 Plugin {
-    type = netstat
-}
-
-Plugin {
-    type = cpu
-}
-
-Plugin {
     type = tray
 }
 
 Plugin {
     type = dclock
     Config {
-        ClockFmt=%R
+        ClockFmt=%H:%M
         TooltipFmt=%A %x
-        BoldFont=0
+        BoldFont=1
+        IconOnly=0
+        CenterText=0
     }
 }
 
 Plugin {
-    type = launchbar
+    type = launchbutton
     Config {
-        Button {
-            id=lxde-logout.desktop
-        }
+        IconPath=/usr/share/faces/kazancas.png
+        Tooltip=Logout
+        Command1=lxde-logout
+        InteractiveUpdates=0
+        InteractiveUpdateRestartInterval=0
     }
 }
 
